### ISSUE TITLE
Permission to modify backpack

### ISSUE THREAD

**Issue by ronangamero** [poqdavid/VirtualTool#14]
So, even with VirtualTool.command.backpackadminread and VirtualTool.command.backpackadminmodify set to true, my admins can't modify others backpacks, only view them. I have * permission and I can't also do that. We're using SpongeAPI: 7.1.0-SNAPSHOT-06016f2 and Spongeforge 1.12.2-2611-7.1.0-BETA-2990

**Comment by poqdavid** [comment]
hmm this is very odd cause this mod is working fine on other servers maybe use LuckPerms?

**Comment by ronangamero** [comment]
We have luck perms, :/

**Comment by poqdavid** [comment]
Do you have an error log or something?

**Comment by ronangamero** [comment]
no, while watching console I don't see any error when trying to do it. I don't know if there's anything I should be looking for in the logs folder too :/ . I just tested it on our new server, that uses a different modpack, I gave virtualtool.command.backpackadminread and virtualtool.command.backpackadminmodify to an account with no other perms on the server, I also added the node virtualtool.command.*, I don't know if that's a thing, but I did just in case. They could open anyone's backpack, but not change it. 
Where should I look if I want to find an error log?
